halo/app/monitor.go

```diff
@@ -167,3 +184,44 @@ func dirSize(path string) (int64, error) {
 
 	return size, nil
 }
+
+// startMonitoringAPI registers metrics and health endpoints.
+func startMonitoringAPI(ctx context.Context, cfg *cmtcfg.Config, cmtNode *node.Node, syncAbort chan error, status *readinessStatus) {
+	mux := http.NewServeMux()
+	mux.Handle("/", promhttp.Handler())
```

Comment:
maybe just add a comment that we align with cometBFT so not registering this at `/metrics` which is the industry standard.

Reply:
Done, I also added `/metrics` too, just in case (the default server exports metrics on any path)

Reply:
nit: update comment, industry standard is `/metrics`, but cometBFT is `/`.